tests/modules/antismash/antismashlitedownloaddatabases/test.yml

```diff
@@ -0,0 +1,8 @@
+- name: antismash antismashlitedownloaddatabases test_antismash_antismashlitedownloaddatabases
+  command: nextflow run tests/modules/antismash/antismashlitedownloaddatabases -entry test_antismash_antismashlitedownloaddatabases -c tests/config/nextflow.config
+  tags:
+    - antismash/antismashlitedownloaddatabases
+    - antismash
+  files:
+    - path: output/antismash/versions.yml
+      md5sum: e2656c8d2bcc7469eba40eb1ee5c91b3
```

Comment:
Missing the actual DB path?

can be simply that it requirs `-path: /<oath>/<to>/<dir>`, don't need a md5sum (as not psossible for directories anyway)